.gitignore

```diff
@@ -21,3 +21,4 @@ picocli.iml
 /src/test/dejagnu.tests/testrun.log
 /src/test/dejagnu.tests/testrun.sum
 /src/test/dejagnu.tests/tmp/
+docs/
```

Comment:
This is not correct; the `docs/` folder should not be in `.gitignore`: the https://picocli.info website is mapped to that directory.